GPU/GPUState.cpp

```diff
@@ -37,6 +37,9 @@ alignas(16) GPUgstate gstate;
 // Let's align this one too for good measure.
 alignas(16) GPUStateCache gstate_c;
 
+// For save state compatibility.
+static int savedContextVersion = 1;
```

Comment:
This should be set to 1 somewhere when a game is started, right? Otherwise it could leak over to the second game you run in a session..

Should really rearchitect the whole emu to avoid all these globals... oh well.

Reply:
Oops, yes, thanks.

Once you have one global, it's kinda hard to avoid more.

-[Unknown]